src/vs/editor/contrib/find/simpleFindWidget.ts

```diff
@@ -44,6 +45,22 @@ export abstract class SimpleFindWidget extends Widget {
 		this._findInput = this._register(new ContextScopedFindInput(null, this._contextViewService, {
 			label: NLS_FIND_INPUT_LABEL,
 			placeholder: NLS_FIND_INPUT_PLACEHOLDER,
+			validation: (value: string): InputBoxMessage | null => {
+				if (value.length === 0) {
+					return null;
+				}
+				if (!this._findInput.getRegex()) {
```

Comment:
What do you think about to put both of those conditions in one _if_ with an _OR_ statement?